src/server.h

```diff
@@ -959,6 +959,7 @@ typedef struct multiState {
                                is possible to know if all the commands have a
                                certain flag. */
     size_t argv_len_sums;    /* mem used by all commands arguments */
+    int alloc_count;         /* total number of multiCmd struct memory reserved. */
```

Comment:
it now occurs to me that maybe we wanna account for that memory in multiStateMemOverhead?

Reply:
I thought `zmalloc_size` of client variable would account for that. I guess that was a wrong assumption.